woodwork/utils.py

```diff
@@ -389,6 +389,28 @@ def get_valid_pearson_types():
     return valid_types
 
 
+def get_valid_spearman_types():
+    """
+    Generate a list of LogicalTypes that are valid for calculating Spearman correlation. Note that
+    index columns are not valid for calculating dependence, but their types may be
```

Comment:
I was under the impression we should be ignoring index columns for all correlation calculations?

Reply:
Yep, they are invalid. This is just warning that their types might still be returned, but we won't use it for calculations